src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -767,27 +770,75 @@ public void testMaxWarnings() throws IOException {
                 .isTrue();
     }
 
-    private static List<String> readWholeFile(File outputFile) throws IOException {
-        return Files.readAllLines(outputFile.toPath(), StandardCharsets.UTF_8);
-    }
+    @Test
+    public final void testExecuteLogOutput() throws Exception {
+        final CheckstyleAntTaskLogStub antTask = new CheckstyleAntTaskLogStub();
+        final URL url = new File(getPath(CONFIG_FILE)).toURI().toURL();
+        antTask.setProject(new Project());
+        antTask.setConfig(url.toString());
+        antTask.setFile(new File(getPath(FLAWLESS_INPUT)));
 
-    private static class CheckstyleAntTaskLogStub extends CheckstyleAntTask {
+        antTask.execute();
 
-        private final List<String> loggedMessages = new ArrayList<>();
+        final Violation auditStartedMessage = new Violation(1,
+                Definitions.CHECKSTYLE_BUNDLE, "DefaultLogger.auditStarted",
+                null, null,
+                getClass(), null);
```

Comment:
this is pretty weird ... it is not Violation ....
can we use:
https://github.com/checkstyle/checkstyle/blob/e0f43fcae85844df469d1c887221d52b5e4e0ea9/src/main/java/com/puppycrawl/tools/checkstyle/DefaultLogger.java#L169-L170


if you took at from testExecuteIgnoredModules, it might be good to fix in all places as first commit.

Reply:
`LocalizedMessage` has `package` access level and is not accessible from this package. But we actually do not need it. We need just a string from the resource bundle. Done.